local_planner/resource/custom_rosconsole.conf

```diff
@@ -1 +1 @@
-log4j.logger.ros.local_planner=ERROR
\ No newline at end of file
+log4j.logger.ros.local_planner=WARN
```

Comment:
We aren't using WARN anywhere else that might get noisy now, are we?

Reply:
now, it is just this paragraph and the missing pointcloud that are WARN